HW02_B_Mimic_starter.Rmd

```diff
@@ -91,7 +110,15 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
-
+mpg$class2 <- factor(mpg$class)
+ggplot(mpg, aes(cty, class)) +
+  geom_point(aes(color = class), position  = position_jitter(width = 0, height = 0.3)) +
+  scale_color_manual("class", values = brewer.pal(7, "Set2")) +

```

Comment:
I don't think it's necessary to indicate again that you want to color by class here since you've already defined that above.

Reply:
I think `scale_color_manual("class")` in this line does not mean the aesthetic, but legend title. But you are right, it is not necessary here since the title is the same as the aesthetic.